backend/kernelCI_app/views/treeDetailsSlowView.py

```diff
@@ -297,6 +317,12 @@ def __processNonBootsTest(self, currentRowData):
             self.testPlatformsWithErrors.add(testPlatform)
             self.testFailReasons[testError] = self.testFailReasons.get(testError, 0) + 1
 
+        if testEnvironmentCompatible is not None:
+            for environment in testEnvironmentCompatible:
+                self.testEnvironmentCompatible[environment] = self.testEnvironmentCompatible.get(
```

Comment:
you can use a `defaultdict` as the original value so it doesn't need this .get with default value